Gruntfile.js

```diff
@@ -131,12 +155,14 @@ module.exports = function(grunt) {
   grunt.loadNpmTasks('grunt-contrib-jshint');
   grunt.loadNpmTasks('grunt-shell');
   grunt.loadNpmTasks('grunt-jstestdriver-phantomjs');
+  grunt.loadNpmTasks('grunt-closurecompiler');
```

Comment:
Should there be a matching change in package.json to specify installation of this package?